### PR TITLE
Add systest conversion xslt to schema_plugins process dir

### PR DESCRIPTION
**Pull request by anguss00**

### CODE REVIEW

File: iso19139.mcp-2.0/process/transform_testing_urls.xsl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    xmlns:gco="http://www.isotc211.org/2005/gco"
+    xmlns:gmd="http://www.isotc211.org/2005/gmd"
+    xmlns:mcp="http://bluenet3.antcrc.utas.edu.au/mcp"
+    exclude-result-prefixes="mcp"
+    version="2.0">
+    
+    <!-- url substitutions to be performed -->
+
+    <xsl:variable name="urlSubstitutions">
+        <substitution match="https?://geoserver-123.aodn.org.au(:443)?" replaceWith="http://geoserver-systest.aodn.org.au"/>
```
Comment (jonescc) [inline iso19139.mcp-2.0/process/transform_testing_urls.xsl]: Need something for geoserver-wps

File: iso19139.mcp-2.0/process/transform_testing_urls.xsl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    <xsl:template match="@*|node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@*|node()"/>
+        </xsl:copy>
+    </xsl:template>
+
+    <!-- change gogoduck urls on systest to correct geoserver -->
+
+    <xsl:template priority="10"
```
Comment (jonescc) [inline iso19139.mcp-2.0/process/transform_testing_urls.xsl]: Not required anymore - we don't have any gogoduck url;s

Comment (jonescc) [inline iso19139.mcp-2.0/process/transform_testing_urls.xsl]: OK - perhaps a better comemnt is why not include this above?

File: iso19139.mcp-2.0/process/transform_testing_urls.xsl (hunk 1 above)
Comment (jonescc) [inline iso19139.mcp-2.0/process/transform_testing_urls.xsl]: Ignore - included below - but perhaps shouldn't be

File: iso19139.mcp-2.0/process/transform_testing_urls.xsl (hunk 2 above)
Comment (anguss00) [inline iso19139.mcp-2.0/process/transform_testing_urls.xsl]: Definitely, makes sense. Will change

**Comment by anguss00** [comment]
Fixed @jonescc

**Pull request merged**